test/benchmarks/driver_bench/src/main.mts

```diff
@@ -199,31 +199,40 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
 }
 
 function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
-  const baselineBench = results.find(r => r.info.test_name === 'cpuBaseline');
-  const pingBench = results.find(r => r.info.test_name === 'ping');
+  const baselineBench = results.find(metricInfoFilterByName('cpuBaseline'));
+  const pingBench = results.find(metricInfoFilterByName('ping'));
 
   assert.ok(pingBench, 'ping bench results not found!');
-  assert.ok(baselineBench, 'baseline results not found!');
-  const pingThroughput = pingBench.metrics[0].value;
-  const cpuBaseline = baselineBench.metrics[0].value;
+  assert.ok(baselineBench, 'cpuBaseline results not found!');
+
+  const cpuBaseline = baselineBench.metrics.find(isMBsMetric);
+  const pingThroughput = pingBench.metrics.find(isMBsMetric);
+
+  assert.ok(cpuBaseline, 'cpu benchmark does not have a MB/s metric');
+  assert.ok(pingThroughput, 'ping does not have a MB/s metric');
 
   for (const bench of results) {
     if (bench.info.test_name === 'cpuBaseline') continue;
+
+    const currentMetric = bench.metrics.find(isMBsMetric);
+    assert.ok(currentMetric, `${bench.info.test_name} does not have a MB/s metric`);
+
     if (bench.info.test_name === 'ping') {
       bench.metrics.push({
         name: 'normalized_throughput',
-        value: bench.metrics[0].value / cpuBaseline,
+        value: currentMetric.value / cpuBaseline.value,
         metadata: {
+          tags: currentMetric.metadata.tags,
           improvement_direction: 'up'
         }
       });
-    }
-    // Compute normalized_throughput of benchmarks against ping bench
-    else {
+    } else {
+      // Compute normalized_throughput of benchmarks against ping bench
       bench.metrics.push({
         name: 'normalized_throughput',
-        value: bench.metrics[0].value / pingThroughput,
+        value: currentMetric.value / pingThroughput.value,
         metadata: {
+          tags: currentMetric.metadata.tags,
```

Comment:
I think this is accurate, but do we not have any tests for the driver spec runner like we do for BSON? Did we ever find out if there is an easy way to see what tags are actually being applied to the metrics?

Reply:
No tests and not an easy way for us since we're not getting the trend chart UI anymore

Reply:
The trend chart UI in EVG is going away (according to the conversations we had about perf.send being deprecated). I assume it's still there because it is leftover